src/NServiceBus.Metrics/CriticalTime/CriticalTimeMetricBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using Metrics;
+using NServiceBus;
+using NServiceBus.Features;
+
+class CriticalTimeMetricBuilder : IMetricBuilder
+{
+    public void WireUp(FeatureConfigurationContext featureConfigurationContext, MetricsContext metricsContext, Unit messagesUnit)
+    {
+        var criticalTimeTimer = metricsContext.Timer("Critical Time", messagesUnit);
```

Comment:
Not sure if Timer actually makes sense here. In the way Metrics.Net defines timer, if we want to use the timer, then we would create the timer, and then define the context, but the using block would be wrapping up the NSB pipeline step for us to utilize it correctly. 
See https://github.com/Recognos/Metrics.NET/wiki/Timers#keeping-track-of-user-values. 
In our case, we already have the pipeline completed event that we are hooking into, so I am not completely sure, how the Metrics timer is helping here.